math/lapack/Portfile

```diff
@@ -2,21 +2,28 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           compilers 1.0
+PortGroup           cmake 1.0
```

Comment:
this should use the latest version of the `cmake` PortGroup, which is "1.1"

Reply:
Done.